Lesson 4 - DOM/code/homework/app.js

```diff
@@ -99,8 +99,12 @@
     // add event listeners observers for both `keydown` and `keyup` events
     // using 'keydown' and 'keyup' functions provided above
 
-    // ??
-    // ??
+    document.addEventListener('keyDown', function (event) {
+        keyDown(event);
+    }, false);
+    document.addEventListener('keyUp', function (event) {
+        keyUp(event);
+    }, false);
```

Comment:
Also you can just pass method name as a second argument to `document.addEventListener` function. You do not need to call it explicit.